amundsen_application/static/js/components/AvatarLabelList/index.tsx

```diff
@@ -0,0 +1,215 @@
+
+import * as React from 'react';
+import ReactDOM from 'react-dom';
+import serialize from 'form-serialize';
+
+import AvatarLabel, { AvatarLabelProps } from '../common/AvatarLabel';
+import { Modal } from 'react-bootstrap';
+
+// TODO: Use css-modules instead of 'import'
+import './styles.scss';
+
+const DEFAULT_ERROR_TEXT = 'There was a problem with the request, please relaod the page.';
+
+enum UpdateMethod {
+  PUT = 'PUT',
+  DELETE = 'DELETE',
+}
+
+export interface DispatchFromProps {
+  onUpdateList: (updateArray: { method: UpdateMethod; id: string; }[], onSuccess?: () => any, onFailure?: () => any) => void;
+}
+
+export interface ComponentProps {
+  errorText?: string | null;
```

Comment:
I think the absence of a value is not treated the same as it being null, so if I want to allow `{ readOnly: true }` -- in which case one doesn't need to pass the `errorText` since that is only used if the component is editable -- I would have to make `errorText` optional. But as with most things Typescript, let me re-run the type checker and see what it complains about.

Reply:
Confirming I need the `errorText` to be marked as optional since we don't want to require it as the component only needs it if it is not `readOnly`.